app/models/aws_s3.rb

```diff
@@ -148,11 +148,11 @@ def put(file_name:, prefix:)
 
   def store(content:, name:, content_type: nil)
     obj = @bucket.object(name)
-    if Rails.env.development? || Rails.env.test?
-      obj.put(body: content, content_type: content_type)
-    else
-      obj.put(body: content, server_side_encryption: 'AES256', content_type: content_type)
-    end
+    args = { body: content }
+    args.merge!(content_type: content_type) if content_type
+    # TODO: document why we skip encryption here
+    args.merge!(server_side_encryption: 'AES256') unless Rails.env.development? || Rails.env.test?
+    obj.put(**args)
```

Comment:
@eanders 
This addresses a failure in `rspec ./spec/models/aws_s3_spec.rb:48` which was added [in this pr](https://github.com/greenriver/hmis-warehouse/pull/4553/files#diff-ded50fd5f0dc4498ae5af0ac5b785601d5319da6b6be9862628d235917ed1a3eR154)

Reply:
[extracted this fix to pr](https://github.com/greenriver/hmis-warehouse/pull/4572)